episodes/chapter5.md

```diff
@@ -0,0 +1,302 @@
+---
+title: "Running a PET analysis using the python client"
+teaching: 
+exercises: 
+---
+
+:::::::::::::::::::::::::::::::::::::: questions
+- How to check the status of a given collaboration within vantage6, programatically, using the Python client?
+- How to request a task based on a given algorithm, programatically, through vantage6's Python client?
+
+:::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Understand the programmatic approach for performing a data analysis through a given scenario.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+# Vantage6 Python client - basic concepts
+
+The vantage6 Python client is a library designed to facilitate interaction with the vantage6 server, to perform various tasks such as creating computation tasks, managing organizations, collaborations, users, and collecting results. It communicates with the server through its REST API, handling encryption and decryption of data for secure operations. The client aims to comprehensively support all aspects of server communication, making it a useful tool for users looking to leverage the full capabilities of the vantage6 platform programmatically.
+
+Creating an instance of the vantage6 Python client is relatively straightforward, requiring only (after installing the corresponding libraries) to define the server connection details: server URL, login credentials, and the organization's private key (if encryption is being used). For simplicity, these settings can be defined on a separate Python module like the following:
```

Comment:
Maybe add that in case the server has 2FA enabled you should also enter the 2FA code.

Reply:
Done